### ISSUE TITLE
Alma.get_data_info : expand_tarfiles=True --> the XXXexternal_ari_l_001_of_001.tar is not expanded

### ISSUE THREAD

**Issue by PhilippeSalome** [astropy/astroquery#2032]
The XXXXexternal_ari_l_001_of_001.tar is a directory (external) like the "products" directory named XXX_001_of_001.tar. 
--> the "products" directory is well expanded with the expand_tarfiles=True in get_data_info but the "external" directory is not expanded (seen as a tar file). An exemple is given below. 
--> Possibilities could be either (i) to expand all .tar when they are directories that contain individual FITS files (products and external) or (ii) add an option with_external=True (like for the auxiliary). The external directory contains very interesting individual FITS files (reprocessed). 

Thanks ! 

```python
In [42]: link_list = Alma.get_data_info(['uid://A001/X87c/X171'], expand_tarfiles=True)
WARNING: AstropyDeprecationWarning: Using the table property is deprecated. Please use se to_table() instead. [pyvo.dal.query]
^[[AWARNING: AstropyDeprecationWarning: Using the table property is deprecated. Please use se to_table() instead. [pyvo.dal.query]

In [43]: for i in range(len(link_list['access_url'])):
    ...:     print(link_list['access_url'][i])
    ...: 
https://almascience.eso.org/dataPortal/member.uid___A001_X87c_X171.README.txt
https://almascience.eso.org/dataPortal/2016.1.00437.S_uid___A001_X87c_X171_001_of_001.tar
https://almascience.eso.org/dataPortal/2016.1.00437.S_uid___A001_X87c_X171_auxiliary.tar
https://almascience.eso.org/dataPortal/2016.1.00437.S_uid___A002_Xb8e961_X4d69.asdm.sdm.tar
**https://almascience.eso.org/dataPortal/2016.1.00437.S_uid___A001_X87c_X171_external_ari_l_001_of_001.tar**
https://almascience.eso.org/dataPortal/member.uid___A001_X87c_X171.NGC5806.CO.flux.fits.gz
https://almascience.eso.org/dataPortal/member.uid___A001_X87c_X171.NGC5806.CO.image.pbcor.fits
https://almascience.eso.org/dataPortal/member.uid___A001_X87c_X171.NGC5806.continuum.flux.fits.gz
https://almascience.eso.org/dataPortal/member.uid___A001_X87c_X171.NGC5806.continuum.image.pbcor.fits

```

**Comment by keflavich** [comment]
@andamian

**Comment by at88mph** [comment]
I think this has been addressed by PR #2493 and/or #2535.  The current `main` tree seems to be OK:

```python
from astroquery.alma import Alma
list_no_expand = Alma.get_data_info(['uid://A001/X87c/X171'], expand_tarfiles=False)
list_expand = Alma.get_data_info(['uid://A001/X87c/X171'], expand_tarfiles=True)

print(len(list_no_expand['access_url']))
print(len(list_expand['access_url']))

```
